src/scss/_vars.scss

```diff
@@ -161,35 +211,44 @@ $email-label-size: var(--label-size2);
 $cta-lg-label-size: var(--label-size1);
 $text-label-size: var(--label-size3);
 $badge-label-size: var(--label-size3);
+$link-button-label-size: var(--link-button-label-size);
```

Comment:
Same question as the heading line height tokens.

Reply:
not sure what you mean here